dto/src/main/scala/com/dwolla/cloudflare/domain/dto/dns/DnsRecordDTO.scala

```diff
@@ -16,8 +16,17 @@ object DnsRecordDTO {
   implicit val dnsRecordDTOCodec: Codec[DnsRecordDTO] = deriveCodec
 }
 
+case class AccountDTO(id: String,
```

Comment:
Here's another couple strings that maybe could be tagged.

Reply:
Addressed.